README.md

```diff
@@ -49,7 +49,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+> Add a link to your deployed project...
```

Comment:
Remove all comments on the markdown file. Comments initially used in the template were meant to guide the user on updating the README file in regards to the content of the project and are now no longer needed.